src/core/ext/filters/client_channel/resolver/dns/c_ares/grpc_ares_wrapper.cc

```diff
@@ -86,8 +88,33 @@ typedef struct grpc_ares_hostbyname_request {
   uint16_t port;
   /** is it a grpclb address */
   bool is_balancer;
+  /** for logging and errors: the query type ("A" or "AAAA") */
+  grpc_core::UniquePtr<char> qtype;
```

Comment:
Please use `std::string` instead of `UniquePtr<char>`.  We're trying to make the latter go away.

Actually, in this particular case, it looks like the value is always created from a string literal, so I don't think there's any need to make a copy of the string.  I think this can just be `const char*`.

Reply:
done, used `const char*`